base/osutils.jl

```diff
@@ -1,20 +1,26 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
-"""
-    OS_NAME
+let UNAME = ccall(:jl_get_UNAME, Any, ())
+    # change some `uname -s` values to "friendly" names
+    # for internal usage
```

Comment:
because the kernel identifier is canonical and unambiguous. these are sort of just random substitutions. I'm more inclined to just drop this renaming and make `Base.OSNAME === Sys.KERNEL`. this does the renaming only because the exported names below use the "friendly" names.